cleanlab/datalab/issue_manager/label.py

```diff
@@ -118,9 +124,36 @@ def _reset(self) -> None:
 
     def find_issues(
         self,
-        pred_probs: np.ndarray,
+        pred_probs: Optional[npt.NDArray] = None,
+        features: Optional[npt.NDArray] = None,
         **kwargs,
     ) -> None:
+        """Find label issues in the datalab.
+
+        Parameters
+        ----------
+        pred_probs :
+            The predicted probabilities for each example.
+
+        features :
+            The features for each example.
+        """
+        if pred_probs is None:
+            if features is None:
+                raise ValueError(
+                    "Either pred_probs or features must be provided to find label issues."
+                )
+            # produce out-of-sample pred_probs from features
+            knn = KNeighborsClassifier(n_neighbors=self.k)
+            knn.fit(features, self.datalab.labels)
+            pred_probs = knn.predict_proba(features)
+
+            encoder = OneHotEncoder()
+            label_transform = self.datalab.labels.reshape(-1, 1)
+            one_hot_label = encoder.fit_transform(label_transform)
+
+            pred_probs = (pred_probs - 1 / self.k * one_hot_label) * self.k / (self.k - 1)
```

Comment:
```suggestion
            pred_probs = np.asarray((pred_probs - 1 / self.k * one_hot_label) * self.k / (self.k - 1))
```

My suggested edit yesterday would make `pred_probs` be of type matrix (as we multiplied it with `one_hot_label` which is a sparse matrix), to make it compatible with the future functions we need to convert it back into an array.